extensions/PythonQt_QtAll/CMakeLists.txt

```diff
@@ -0,0 +1,103 @@
+project(QtAll LANGUAGES CXX)
+
+set(CMAKE_AUTOMOC ON)
+
+file(GLOB SOURCES *.h *.cpp)
+
+if(BUILD_SHARED_LIBS)
+    add_library(${PROJECT_NAME} SHARED)
+else()
+    add_library(${PROJECT_NAME} STATIC)
+endif()
+
+target_sources(${PROJECT_NAME} PRIVATE
+    ${SOURCES}
+    ${GENERATE_SOURCES}
+)
+
+target_link_libraries(${PROJECT_NAME} PUBLIC
+    Core
+)
+
+target_include_directories(${PROJECT_NAME} PUBLIC ${CMAKE_CURRENT_LIST_DIR})
+
+target_compile_definitions(${PROJECT_NAME} PRIVATE PYTHONQT_QTALL_EXPORTS)
+
+list(APPEND QTMODULES Core Gui Svg Sql Network OpenGL Xml XmlPatterns Multimedia Qml Quick UiTools WebEngineWidgets WebKit)
```

Comment:
I think this list shall be different for Qt5 and Qt6.

Reply:
I think as long as there are no modules with the same name but different functionality in Qt 5 and Qt 6, there shouldn't be a problem.